src/server/firefox.ts

```diff
@@ -185,3 +189,129 @@ export class Firefox implements BrowserType<FFBrowser> {
   }
 }
 
+function wrapTransportWithWebSocket(transport: ConnectionTransport, port: number): string {
+  const server = new ws.Server({ port });
+  const guid = platform.guid();
+  const idMixer = new SequenceNumberMixer<{id: number, socket: ws}>();
+  const pendingBrowserContextCreations = new Set<number>();
+  const pendingBrowserContextDeletions = new Map<number, string>();
+  const browserContextIds = new Map<string, ws>();
+  const sessionIds = new Map<string, ws>();
+  const sockets = new Set<ws>();
+
+  transport.onmessage = message => {
+    if (typeof message.id === 'number') {
+      // Process command response.
+      const value = idMixer.take(message.id);
```

Comment:
Could you please do `const seqNum = message.id;` and use that below, for clarity?

Reply:
Done